### PR TITLE
Added decent styling to the webpage.

### PR DESCRIPTION
**Pull request by codedmachine111**
Added a decent look to the page by importing better fonts, styling the button, search-bar, and text. Fixes issue #2 .

### CODE REVIEW

**Comment by codedmachine111** [comment]
Hello @neupanedipen . I have added a decent styling to the webpage by importing decent fonts and enhancing the buttons, search bar and text. Kindly review my PR.

File: public/index.html [hunk 1]
```diff
@@ -25,6 +25,15 @@
   <title>YouTube Playlist Duration Calculator</title>
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Urbanist:wght@100;200;300;400&display=swap" rel="stylesheet">
+  <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap" rel="stylesheet" />
+  <script type="text/javascript" src="http://code.jquery.com/jquery-latest.min.js"></script>
```
Comment (neupanedipen) [inline public/index.html]: Since you're using http for jquery it's giving a mixed content error. Maybe it'll get resolved if you change it to https.

**Comment by codedmachine111** [comment]
Hello @neupanedipen . I apologize for the error. I was not using jQuery anyway, hence I took it out from the code. Kindly review my PR, again.

**Comment by neupanedipen** [comment]
Done. Thank you for contributing.

**Pull request merged**